app/api/test-slack/route.ts

```diff
@@ -0,0 +1,35 @@
+import { NextResponse } from "next/server";
+import { z } from "zod";
+import { postSlackMessage } from "@/server/index";
+
+const testRequestSchema = z.object({
+  channel: z.string().min(1),
+  message: z.string().min(1)
+});
+
+export async function POST(request: Request) {
+  try {
+    const body = await request.json();
+    const { channel, message } = testRequestSchema.parse(body);
+
+    const channelName = channel.startsWith("#") ? channel : `#${channel}`;
+    
+    const slackResponse = await postSlackMessage({
```

Comment:
Instead of calling post Slack Message, we're actually mocking a slack message being posted on slack and firing the webhook.

So in general, we should be calling the slack webhook instead